### DISCUSSION TITLE
Sequential printer and skirt.

### DISCUSSION THREAD

**Discussion by gjelsoe** [supermerill/SuperSlicer#2100] [General]
I've encountered some unwanted behavior when using a skirt on each item in a sequential printing.

As I've seen so far, when one item is done and starts printing the skirt for next item, it actually lowers the nozzle inside of the area where the item who is next in line to get printed, then moves to the left and starts printing the skirt. When using Filament that likes to ooze, you will get unwanted material in your print and the print quality suffers.

Avoid crossing perimeters was checked to this print and was crossed anyway.

Is't possible make it print the skirt without crossing print area of the new item, maybe calculating a new start position of the skirt and also save some travel movement

**Comment by supermerill** [comment]
mmm
if you set your `lift_min` higher than the first layer object height, you'll have the desired behaviour.
I think I understand why it doesn't work when it's lower, the codepath is skipped.

**Reply by gjelsoe** [reply]
It seems like the start pos for the first skirt is correct but when is comes to the next skirt/part, it gets the wrong start pos.

**Reply by gjelsoe** [reply]
Will try lift_min.

**Reply by gjelsoe** [reply]
The travel path is the same, even with lift_min set to 0.4 and layer height is 0.28

**Reply by supermerill** [reply]
try with a valuie for lift_min higher than your object height. Like 22mm

**Reply by gjelsoe** [reply]
No change
![odd](https://user-images.githubusercontent.com/36234524/147011207-cd4a777f-bc88-433d-ab08-b04db5405732.PNG)

**Reply by gjelsoe** [reply]
Should have been like this:
![odd_correct](https://user-images.githubusercontent.com/36234524/147011461-352acee7-bbde-489f-939d-7060ab759dba.PNG)

**Comment by supermerill** [comment]
![image](https://user-images.githubusercontent.com/6536403/147076785-cc7e5d4c-5ca4-464d-9bad-1048a8d4bbab.png)
[min_lift.zip](https://github.com/supermerill/SuperSlicer/files/7761896/min_lift.zip)

**Reply by gjelsoe** [reply]
Got it working now :)

But ended up with editing the G-Code to make it behave as i wanted..
